src/webui/www/private/scripts/client.js

```diff
@@ -250,11 +263,23 @@ window.addEvent('load', function() {
 
     // After showing/hiding the toolbar + status bar
     let showSearchEngine = LocalPreferences.get('show_search_engine') !== "false";
+    let showRssReader = LocalPreferences.get('show_rss_reader') !== "false";
+
+    if (!showSearchEngine && !showRssReader) {
+        // hide tabs
+        $('mainWindowTabs').addClass('invisible');
+    }
+
     if (!showSearchEngine) {
+        $('searchTabLink').addClass('invisibleImp');
```

Comment:
What does `Imp` stand for in `invisibleImp`? Maybe there's a more descriptive name that can be used instead.

Reply:
Imp stands for the css `!important` modifier. But since it's bad form anyway I changed things to make the normal `invisible` class work.